pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/BuildWheelsTask.groovy

```diff
@@ -97,13 +107,7 @@ class BuildWheelsTask extends DefaultTask implements SupportsWheelCache, Support
     /**
      * Will return true when the package should be excluded from being installed.
```

Comment:
is this comment still accurate?

Reply:
Yes. It still documents what the intended usage of this filter is. It's still a correct documentation for a public field.